README.rst

```diff
@@ -1,34 +1,75 @@
-|Build Status| |Codecov| |npm_version| |npm_downloads| |license| |semantic-release|
+|Build Status| |Codecov| |npm_version| |license|
 
 frontend-app-profile
 ====================
 
-Please tag **@edx/arch-fed** on any PRs or issues.
+This is a micro-frontend application responsible for the display and
+updating of user profiles. Please tag **@edx/arch-fed** on any PRs or
+issues.
 
-Introduction
-------------
+----------
 
-React app for user account management.
+Development
+-----------
 
-Important Note
---------------
+Start Devstack
+^^^^^^^^^^^^^^
 
-The production Webpack configuration for this repo uses `Purgecss <https://www.purgecss.com/>`_ 
-to remove unused CSS from the production css file. In webpack/webpack.prod.config.js the Purgecss
-plugin is configured to scan directories to determine what css selectors should remain. Currently
-the src/ directory is scanned along with all @edx/frontend-component* node modules and paragon.
-If you add and use a component in this repo that relies on HTML classes or ids for styling you
-must add it to the Purgecss configuration or it will be unstyled in the production build. 
+To use this application `devstack <https://github.com/edx/devstack>`__
+must be running and you must be logged into it.
+
+-  Start devstack
+-  Log in (http://localhost:18000/login)
+
+Start the development server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this project, install requirements and start the development server
+by running:
+
+.. code:: bash
+
+   npm install
+   npm start # The server will run on port 1995
+
+Once the dev server is up visit http://localhost:1995/u/staff.
+
+----------
+
+
+Configuration and Deployment
+----------------------------
+
+This MFE is configured via node environment variables supplied at build time. See .env.development for the list of required environment variables. Example build syntax:
+
+.. code:: bash
+
+   NODE_ENV=development ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload' npm run build
```

Comment:
```suggestion
   NODE_ENV=production ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload' npm run build
```

And might want to point out you _need_ to supply all of the env variables, and that this is just a simple example of the syntax...